docker/mock-blockchain.Dockerfile

```
@@ -0,0 +1,39 @@
+# Dockerfile for Mock Blockchain Services
+FROM oven/bun:1.2.0-alpine AS base
+
+# Set working directory
+WORKDIR /app
+
+# Install system dependencies
+RUN apk add --no-cache \
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)